ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -125,7 +125,7 @@ router bgp {{ router_bgp.as }}
 {%         if peer_group.ebgp_multihop is arista.avd.defined %}
    neighbor {{ peer_group.name }} ebgp-multihop {{ peer_group.ebgp_multihop }}
 {%         endif %}
-{%         if peer_group.password is arista.avd.defined %}
+{%         if peer_group.password is arista.avd.defined and peer_group.password is arista.avd.valid_password(passwd_type="bgp", key=peer_group.name) %}
```

Comment:
We should not do input validation in the templates. This should be covered by the schema.